### PR TITLE
Add floating.js w/ npm auto-update via single package.json

### LINKED ISSUE #9848: [Request] Add floating.js
**Library name:** floating.js
**Git repository url:** https://github.com/Haroenv/floating.js/
**npm package url(optional):** floating.js
**License(s):** Apache-2.0
**Official homepage:** https://haroen.me/floating.js/

In this issue, we are going to host a new web front-end library on cdnjs, so that web developer can easily use it on our free CDN without additional download/upload process, if you'll like to work locally, I'll suggest you start with sparseCheckout: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md, since there are too many files in this repo, without sparseCheckout, the whole process will be super slow, using git command line can learn more skills and experience, if you don't like that, instead, you can also just do it on GitHub, it's more convenient, but it may be a little bit harder to fix problems on GitHub gui.

Here are the docs about how to add a library:
https://github.com/cdnjs/cdnjs#adding-a-new-or-updating-an-existing-library
https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md#d-adding-a-new-library

We'll also ask to setup auto-update config so that the added libraries can be updated to the latest version easily:
https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

For beginners who interested in join and help add this library, please feel free to tell me what kind of help you'll need, we'll help you send the valid pull request, thanks.

### PR DESCRIPTION
**Pull request by dakshshah96**
close #9848, cc @Haroenv

Pull request for issue: #9848

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Haroenv/floating.js/
 * Official website (optional, not the repository): https://haroen.me/floating.js/
 * NPM package url (optional): https://www.npmjs.com/package/floating.js
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 3
   - Count of stars: 150
   - Count of forks: 11
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 34
   - Downloads in the last month: 246

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/floating.js/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Float a number of things up on a page (hearts, flowers, ...)",
+  "homepage": "https://haroen.me/floating.js/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroenv/floating.js.git"
+  },
+  "author": "Haroen Viaene",
+  "license": "Apache-2.0",
+  "filename": "floating.js",
+  "keywords": [
```
Comment (extend1994) [inline ajax/libs/floating.js/package.json]: Please modify the keywords to let users be able to find the library in a easier way. You can test if the keywords are good by googling them. If GitHub or homepage displays on first page of search results, then it's good. For instance, `javascript`, `element`, `float`, `infinitely`.

Comment (hare1039) [inline ajax/libs/floating.js/package.json]: I think we can add `v1.0.1` later on.

Comment (Haroenv) [inline ajax/libs/floating.js/package.json]: I don't think it's worth to care about those two old versions that had another patch just after, right?

Comment (PeterDaveHello) [inline ajax/libs/floating.js/package.json]: @dakshshah96 can you please update keywords like https://github.com/Haroenv/floating.js/blob/c10660a50ee0679a0aaed0867376529a23af9fca/package.json#L18-L23

**Comment by dakshshah96** [comment]
Is this PR good to merge? @PeterDaveHello

**Comment by pvnr0082t** [comment]
>@dakshshah96 can you please update keywords like https://github.com/Haroenv/floating.js/blob/c10660a50ee0679a0aaed0867376529a23af9fca/package.json#L18-L23

As @PeterDaveHello said, I think you should update keywords first.

**Comment by dakshshah96** [comment]
@pvnr0082t I overlooked that, my bad! On it now.

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍

**Pull request merged**